sites/hcinnovationgroup.com/server/templates/website-section/top-100.marko

```diff
@@ -0,0 +1,96 @@
+import hierarchyAliases from "@base-cms/marko-web/utils/hierarchy-aliases";
+import queryFragment from "../../graphql/fragments/top100-landing-page";
+import GAM from "../../../config/gam";
+
+$ const formatCurrentcy = function(number) {
+  return '$' + new Intl.NumberFormat().format(number)
+};
+
+$ const { id, alias, name, pageNode } = data;
+
+<marko-web-website-section-page-layout id=id alias=alias name=name>
+  <@head>
+    <marko-web-gtm-website-section-context|{ context }| alias=alias>
+      <marko-web-gtm-push data=context />
+    </marko-web-gtm-website-section-context>
+    <marko-web-resolve-page|{ data: section }| node=pageNode>
+      $ const aliases = hierarchyAliases(section);
+      $ const adSlots = {
+          "gpt-ad-lb1":   GAM.getAdUnit({ name: "lb1", aliases }),
+        }
+       <marko-web-gam-slots slots=adSlots />
+    </marko-web-resolve-page>
+  </@head>
+  <@above-container>
+    <marko-web-resolve-page|{ data: section }| node=pageNode>
+      $ const aliases = hierarchyAliases(section);
+      <marko-web-gam-out-of-page-ad ...GAM.getAdUnit({ name: "wa", aliases }) />
+      <marko-web-gam-out-of-page-ad ...GAM.getAdUnit({ name: "reskin", aliases }) />
+    </marko-web-resolve-page>
+  </@above-container>
+  <@page>
+    <marko-web-gam-display-ad id="gpt-ad-lb1" modifiers=["top-of-page"] />
+    <marko-web-resolve-page|{ data: section }| node=pageNode>
+    $ const aliases = hierarchyAliases(section);
+      <marko-web-page-wrapper class="mb-block">
+        <@section>
+          <div class="row">
+            <div class="col">
+              <default-theme-website-section-breadcrumbs section=section display-self=false />
+              <marko-web-website-section-name tag="h1" class="page-wrapper__title" obj=section />
+              <if(section.description)>
+                <p class="page-wrapper__deck">$!{section.description}</p>
+              </if>
+              <hr>
+            </div>
+          </div>
+        </@section>
+        <@section>
+          <div class="row">
+            <div class="col-lg-12 mb-block">
+              <marko-web-query|{ nodes }|
+                name="website-scheduled-content"
+                params={ sectionId: id, includeContentTypes: ["Top100"], limit: 100, queryFragment }
+              >
+                <!-- @todo add support for sort by rank -->
+                $ nodes = nodes.sort((a,b) => (a.rank > b.rank) ? 1 : ((b.rank > a.rank) ? -1 : 0));
```

Comment:
Do not reassign the `nodes` value, instead do `$ const sorted = nodes.sort(...)`